airflow/cli/commands/variable_command.py

```diff
@@ -61,8 +61,8 @@ def variables_get(args):
 @cli_utils.action_cli
 @providers_configuration_loaded
 def variables_set(args):
-    """Create new variable with a given name and value."""
-    Variable.set(args.key, args.value, serialize_json=args.json)
+    """Create new variable with a given name, value and description."""
+    Variable.set(args.key, args.value, args.description or None, serialize_json=args.json)
```

Comment:
```suggestion
    Variable.set(args.key, args.value, args.description, serialize_json=args.json)
```

I’d assume the user would have a use case if they explicitly supply an empty string.

Reply:
Ack, I will include this in the next revision as well

Reply:
This is fixed in next rev